docs/docsite/rst/community/steering/community_topics_workflow.rst

```diff
@@ -4,121 +4,124 @@
-
-  * Create a new post in the topic.
-
-  * Click the ``gear`` button in the composer and select ``Build Poll``.
+------------
 
-  * Click the ``gear`` in the Poll Builder for advanced mode.
+The Committee person:
 
-  * Set up the options (generally this will be Single Choice but other poll types can be used).
+* Depending on the topic's complexity, 1-2 weeks after the discussion was opened, formulates vote options based on the prior discussion and gives participants reasonable amount of time to propose changes to the options (no longer than a week).
+* Summarizes the options in a comment and also establishes a date when the vote begins and ends if there are no objections about the options.
+* In the vote date, starts the vote and establishes its end date, which is $CURRENT_DATE plus:
 
-  * Title it "Steering Committee vote" and "Limit voting" to the ``Steering Committee``.
+  * 7 days: simple cases
+  * 14 days: should not be more than that
+  * 21 days: only used in cases like common holiday seasons when the majority of the Committee out of participation
+* Labels the topic with the ``active-vote`` tag.
+* Adds ``[Vote ends on $YYYY-MM-DD]`` to the beginning of the topic's description.
 
-  * Do not set the close date because it cannot be changed later.
-
-  * Results should be "Always Visible" unless there is some good reason for the SC votes not to be public.
-
-  * Submit the poll (the BBcode will appear in the post) and then repeat the above for the second poll.
-
-    * The title should be "Community vote".
+The vote always consists of two polls: one for the Steering Committee, one for everyone else. To create a vote in a topic:
 
+  * Create a new post in the topic.
+  * Click the ``gear`` button in the composer and select ``Build Poll``.
+  * Click the ``gear`` in the ``Poll Builder`` for advanced mode.
+  * Set up the options (generally this will be ``Single Choice`` but other poll types can be used).
+  * Title it "Steering Committee vote" and ``Limit voting`` to the ``@SteeringCommittee``.
+  * Do NOT set the close date because this cannot be changed later.
+  * Results should be ``Always Visible`` unless there is some good reason for the SC votes not to be public.
+  * Submit the poll (the BBcode will appear in the post):
+  * Repeat the above for the second poll:
+
+    * Title should be "Community vote".
     * No group limitation.
 
 Voting result stage
-^^^^^^^^^^^^^^^^^^^
-
-* The day after the last day of the vote, the Committee person:
-
-  * Closes the polls.
+-------------------
 
-  * Removes the ``active-vote`` tag.
-
-  * Add a comment that the vote ended.
-
-  * Changes the beginning of the topic's description to ``[Vote ended]``.
-
-  * Creates a summary comment declaring the vote result.
-
-* The vote result and final decision are announced via the `Bullhorn newsletter <https://forum.ansible.com/c/news/bullhorn/17>`_.
+The next day after the last day of the vote, the Committee person:
```

Comment:
Is this the day after the last day or two days after the last day?

Reply:
Let's simplify. We actually did this as i changed it